src/views/download-file.njk

```diff
@@ -0,0 +1,32 @@
+{% extends 'layout.njk' %}
+
+{% block pageTitle %}
+  {% if errorList.length > 0 %}
+    Error:
+  {% endif %}
+  Sign in
+{% endblock %}
+{% block beforeContent %}
```

Comment:
That seems unneeded, layout template comes with phase banner configured with right feedback link.